api/api-invoices/src/invoices/toSheetLine.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable prefer-destructuring */
+const { omit } = require('lodash')
```

Comment:
loadash, well, well, well... 😛

Reply:
we are server side 
so the bundle size is not really a high importance